app/views/questionnaire.py

```diff
@@ -457,6 +460,20 @@ def _get_front_end_navigation(answer_store, current_location, metadata):
         return None
 
 
+def get_page_title_for_location(schema_json, current_location):
+    block = SchemaHelper.get_block_for_location(schema_json, current_location)
+    if block['type'] == 'Interstitial':
+        group = SchemaHelper.get_group(schema_json, current_location.group_id)
+        page_title = '{group_title} - {schema_title}'.format(group_title=group['title'], schema_title=schema_json['title'])
+    elif block['type'] == 'Questionnaire':
+        first_question = SchemaHelper.get_questions_for_block(block)[0]
```

Comment:
As it's the only place we get the first question does it need to be refactored out if get_questions_for_block is a list generator?